docs/api.md

```diff
@@ -3,61 +3,101 @@ id: api
 title: API
 sidebar_label: API
 ---
- 
+
 If a user visits your page and has Web Monetization enabled, your site can
 detect that Web Monetization is enabled.
 
-You can access Web Monetization-related events and state through a Javascript
+You can access Web Monetization related events and state through a Javascript
 API on `document.monetization`.
 
 ## Javascript API
 ```ts
 document.monetization: EventTarget
-document.monetization.state: string
+document.monetization.state: 'stopped' | 'pending' | 'started'
```

Comment:
Are we sure we don't want to just leave `pending` as is ( essentially stopped ) and add an `initializing` state (essentially pending ) 

At some point we'll have to follow up with some breaking changes to match the W3C standard, so it would just be a temporary "strangeness"

If we are concerned about breaking things in the short term, that would help minimize changes.

Reply:
OK then